src/marqo/tensor_search/tensor_search_logging.py

```diff
@@ -1,9 +1,23 @@
 import logging
-
+from marqo.tensor_search.utils import read_env_vars_and_defaults
+from marqo.errors import EnvVarError
 def get_logger(name):
-    logging.basicConfig() 
+    logging.basicConfig()
     logger = logging.getLogger(name)
-    logger.setLevel(logging.INFO)
+
+    log_level = read_env_vars_and_defaults("MARQO_LOG_LEVEL").lower()
+    if log_level == "warning":
+        logger.setLevel(level=logging.WARNING)
+    elif log_level == "info":
+        logger.setLevel(level=logging.INFO)
+    elif log_level == "debug":
+        logger.setLevel(level=logging.DEBUG)
+    elif log_level == "error":
+        logger.setLevel(level=logging.ERROR)
+    else:
+        raise EnvVarError(f"The provided {log_level} is not supported."
+                          f"The {log_level} should be one of `error`, `warning`, `info`, `debug`."
+                          f"Check https://docs.marqo.ai/0.0.13/Advanced-Usage/configuration/ for more info.")
```

Comment:
Please surround the the user's log level with backticks:
```
`{log_level}`
```
Also, we think about how this looks printed out, it looks odd: 
```
The provided MY_CUSTOM_LEVEL is not supported. The MY_CUSTOM_LEVEL should be one of `error`, `warning`, `info`, `debug`....
```
The user needs to know the problem is with the log level. Can you also make explicit reference to the env var the user needs to correct if they get this error?
```
The provided log level `MY_CUSTOM_LEVEL`, set by the environment variable `MARQO_LOG_LEVEL` is not supported. The log level should be one of `error`, `warning`, `info`, `debug`. 
```

Reply:
thanks! customer first!

Reply:
done